addons/website_sale/static/src/js/website_sale_utils.js

```diff
@@ -34,6 +34,7 @@ const cartHandlerMixin = {
             route: "/shop/cart/update_json",
             params: params,
         }).then(async data => {
+            sessionStorage.setItem('website_sale_cart_quantity', data.cart_quantity);
```

Comment:
import from `@web/core/browser/browser` and use `sessionStorage` from that object instead of the one of the browser directly